go/vt/vtadmin/api.go

```diff
@@ -222,15 +223,7 @@ func (api *API) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	dynamicAPI := &API{
-		clusters:   api.clusters,
-		clusterMap: api.clusterMap,
-		router:     api.router,
-		serv:       api.serv,
-		authz:      api.authz,
-		options:    api.options,
-	}
-
+	var dynamicAPI *API
```

Comment:
if we're able to unpack the cookie, this will become a second API with its own `clusters` and `clusterMap` structures. if we get to the bottom and we're still `nil`, we use the original api, with the original structures and mutex.